src/features/customer/cart/view/CartView.js

```diff
@@ -35,6 +54,15 @@ const CartView = () => {
       ) : (
         <CartEmpty />
       )}
+
+      <Modal
+        visible={isLoginVisible}
+        onCancel={() => setIsLoginVisible(false)}
+        footer={null}
+      >
+        <AuthenticationModal isOpen={isLoginVisible} onClose={handleLoginSuccess} />

```

Comment:
onClose'a handleLoginSuccess vermeyelim kullanici login olmasa da bu calisacak, popup acildiktan sonra direkt kapatarak deneyebilirsin bu case'i

Reply:
Istersen iki tur fonksiyon verelim bu modal'e birisi onCancel digeri onAuth gibi bu fonksiyonunu onAuth'a verebilirsin bu sekilde

Reply:
hallettim ama üç tür fonksiyon var, istersen bak olmazsa dediğin gibi yapıyım